### PR TITLE
Simplify `F.floor` test

### PR DESCRIPTION
**Pull request by hikjik**
Simplifies test_floor using testing.FunctionTestCase
Refer to [#6071](https://github.com/chainer/chainer/issues/6071)

### CODE REVIEW

File: tests/chainer_tests/functions_tests/math_tests/test_floor.py [hunk 1]
```diff
@@ -1,86 +1,53 @@
...
+class TestFloor(testing.FunctionTestCase):
+
+    skip_backward_test = True
+    skip_double_backward_test = True
+
+    def generate_inputs(self):
+        while True:
+            x = numpy.random.uniform(
+                -10.0, 10.0, self.shape).astype(self.dtype)
+            if (numpy.abs(x - numpy.round(x)) > 1e-2).all():
```
Comment (Crissman) [inline tests/chainer_tests/functions_tests/math_tests/test_floor.py]: Please comment on why this distance check is required.

Comment (hikjik) [reply tests/chainer_tests/functions_tests/math_tests/test_floor.py]: This distance check is used for stability. Please take a look here [#3427](https://github.com/chainer/chainer/pull/3427)

Comment (Crissman) [reply tests/chainer_tests/functions_tests/math_tests/test_floor.py]: Understood. Thanks.

**Comment by Crissman** [comment]
LGTM. Jenkins, test this please.

**Comment by Crissman** [comment]
Jenkins failure unrelated. Merging.

**Pull request merged**